src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1614,6 +1624,24 @@ class DocBaseViewer extends BaseViewer {
     handleAnnotationCreatorChangeEvent({ status, type }) {
         this.processAnnotationModeChange(this.annotationControlsFSM.transition(status, type));
     }
+
+    handleFSMChange = () => {
```

Comment:
We should probably use `.bind` in the constructor to stay consistent with all other handlers in this class.